lib/source/metadata.js

```diff
@@ -53,90 +50,55 @@ class Metadata extends Source.Polling(Parser) { // eslint-disable-line new-cap
    * @returns {string}
    * @private
```

Comment:
Since the function name lost its underscore, I'm assuming it's now public. Do we want to drop the `@private` status from the doc comment?